.devcontainer/scripts/initialize.sh

```
@@ -11,5 +11,24 @@ if [ ! -f "${CLASP_RC}" ]; then
+fi
+
+# Make sure global node modules are accessible
+if ! grep -q "export PATH=\$PATH:\$HOME/.npm-global/bin" "${HOME}/.bashrc"; then
+    mkdir -p "${HOME}/.npm-global"
+    npm config set prefix "${HOME}/.npm-global"
+    echo 'export PATH=$PATH:$HOME/.npm-global/bin' >> "${HOME}/.bashrc"
+    echo "Updated PATH to include global npm binaries"
+fi
+
+# Make directories writable
+mkdir -p /workspaces/$(basename ${PWD})/build
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)